spec/danger_plugin_spec.rb

```diff
@@ -32,25 +32,30 @@ module Danger
         end
 
         it 'handles a known rubocop report' do
-          allow(@rubocop).to receive(:`).with('bundle exec rubocop -f json spec/fixtures/ruby_file.rb').and_return(@rubocop_response)
+          allow(@rubocop).to receive(:`)
+            .with('bundle exec rubocop -f json spec/fixtures/ruby_file.rb')
+            .and_return(@rubocop_response)
 
           # Do it
-          @rubocop.run("spec/fixtures/*.rb")
+          @rubocop.run('spec/fixtures/*.rb')
 
           output = @rubocop.status_report[:markdowns].first
 
           expect(output).to_not be_empty
 
           # A title
-          expect(output).to include("Rubocop violations")
+          expect(output).to include('Rubocop violations')
           # A warning
-          expect(output).to include("ruby_file.rb | 13 | Don't do that!")
+          expect(output).to include("ruby_file.rb | 13   | Don't do that!")
```

Comment:
Ah I see, so this is probably a consequence of using terminal-table with "line" for a heading, cool.